addons/website/views/snippets/s_blockquote.xml

```diff
@@ -20,55 +34,81 @@
     </blockquote>
 </template>
 
-<template id="s_blockquote_options" inherit_id="website.snippet_options">
-    <xpath expr="." position="inside">
-        <!-- Color and images -->
-        <t t-call="website.snippet_options_background_options">
-            <t t-set="selector" t-value="'.s_blockquote'"/>
-            <t t-set="with_colors" t-value="True"/>
-            <t t-set="with_images" t-value="True"/>
-            <t t-set="with_shapes" t-value="True"/>
-            <t t-set="with_gradients" t-value="True"/>
-            <t t-set="with_color_combinations" t-value="True"/>
-        </t>
+<template id="s_blockquote_options">
+    <div t-if="not disable_blockquote_width" t-att-data-selector="selector" t-att-data-target="target" t-att-data-exclude="exclude">
+        <we-select string="Width">
+            <we-button data-select-class="w-25">25%</we-button>
+            <we-button data-select-class="w-50">50%</we-button>
+            <we-button data-select-class="w-75">75%</we-button>
+            <we-button data-select-class="w-100" data-name="so_width_100">100%</we-button>
+        </we-select>
+        <we-button-group string="Alignment" class="o_we_sublevel_1" data-dependencies="!so_width_100">
+            <we-button class="fa fa-fw fa-align-left" title="Left" data-select-class="me-auto"/>
+            <we-button class="fa fa-fw fa-align-center" title="Center" data-select-class="mx-auto"/>
+            <we-button class="fa fa-fw fa-align-right" title="Right" data-select-class="ms-auto"/>
+        </we-button-group>
+    </div>
+    <!-- Color and images -->
+    <t t-call="website.snippet_options_background_options">
+        <t t-set="selector" t-value="selector"/>
+        <t t-set="exclude" t-value="exclude"/>
+        <t t-set="target" t-value="target"/>
+        <t t-set="with_colors" t-value="True"/>
+        <t t-set="with_images" t-value="True"/>
+        <t t-set="with_shapes" t-value="True"/>
+        <t t-set="with_gradients" t-value="True"/>
+        <t t-set="with_color_combinations" t-value="True"/>
+    </t>
 
-        <!-- Layout -->
-        <div data-selector=".s_blockquote">
-            <we-range string="Edge Spacing" data-select-class="p-1|p-2|p-3|p-4|p-5"/>
-            <we-select string="Decoration">
-                <we-button data-select-class="s_blockquote_default">None</we-button>
-                <we-button data-select-class="s_blockquote_with_line" data-name="blockquote_with_line_opt">Left line</we-button>
-                <we-button data-select-class="s_blockquote_with_icon">Icon</we-button>
-            </we-select>
-        </div>
+    <!-- Layout -->
+    <div t-att-data-selector="selector" t-att-data-target="target" t-att-data-exclude="exclude">
+        <we-range string="Edge Spacing" data-select-class="p-1|p-2|p-3|p-4|p-5"/>
```

Comment:
The blockquote currently comes with an enforced `gap-4` class, which works well for the **current** intended use (e.g., large quote cards in a carousel). However, this new snippet introduces use cases for different scenarios. It would be useful to make the gap editable, especially when the review is brief or there’s no review at all.
What do you think?

|`gap-4` | `gap-0` |
|--------|--------|
| <img width="637" alt="image" src="https://github.com/user-attachments/assets/56d8f498-bfcf-4e2d-9dd6-52a79e8335ac"> | <img width="324" alt="image" src="https://github.com/user-attachments/assets/524ae8a1-beed-4849-bea6-b0b48e901186"> |

Reply:
Not sure about this one, the gap looks fine IMO. I think you still have the gap coming from the empty line if your screenshot, which makes it look super large (?)
<img width="465" alt="image" src="https://github.com/user-attachments/assets/6d78421d-0d1f-4fed-8565-2c6626b26b24">

Reply:
Ok, as discussed at the office, let's ignore the scenario for now  👍